src/register/controller.ts

```diff
@@ -16,6 +16,15 @@ class RegistrationController extends Controller {
 
   async post(ctx: Context) {
 
+    const userPassword = ctx.request.body.password;
```

Comment:
Given that you're putting the password in its own variable here, then line 44 should also use it